package.json

```diff
@@ -30,9 +30,11 @@
   "dependencies": {
     "async": "^0.9.x",
     "jsonld": "^0.4.5",
+    "mz": "^2.6.0",
```

Comment:
Since you're only using `mz` in tests, this should go in devDependencies. Also, I'd prefer it if you used `fs-extra` instead.